portfolio/src/main/webapp/index.html

```diff
@@ -1,56 +1,54 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
+    s<meta charset="UTF-8">
```

Comment:
[3.1.2](https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity): Use valid HTML where possible.

```suggestion
    <meta charset="UTF-8">
```

This is invalidating stuff `<head>` if you use the [W3C HTML validator](https://validator.w3.org/nu/).